kernel/kselftest.py.data/kselftest.yaml

```diff
@@ -2,8 +2,8 @@ setup:
     component: !mux
         vm:
             comp: "vm"
-        power:
-            comp: "powerpc"
+#        power:
+#            comp: "powerpc"
```

Comment:
why not remove it ?

Reply:
@abdhaleegit as we have an issue while running sigfuzz test it crashes the system we are waiting for fix
@naveen538  please add a comment as well

Reply:
@abdhaleegit As we are waiting for fix so commented powerpc.